managedplugin/download_test.go

```diff
@@ -44,11 +44,11 @@ func TestDownloadPluginFromCloudQueryIntegration(t *testing.T) {
 		wantErr  bool
 		typ      PluginType
 	}{
-		{testName: "should download test plugin from cloudquery registry", team: "cloudquery", plugin: "test", version: "v3.1.11", typ: PluginSource},
+		{testName: "should download test plugin from cloudquery registry", team: "cloudquery", plugin: "azuredevops", version: "v3.0.12", typ: PluginSource},
```

Comment:
The test plugin is private so we can't download it without an auth token. It worked before due to a bug we fixed